examples/cms-kontent/components/image.js

```diff
@@ -0,0 +1,32 @@
+import NextImage from 'next/image'
+import { transformImageUrl } from '@kentico/kontent-delivery'
+
+const KONTENT_ASSET_HOSTNAME_REGEX = /.kc-usercontent.com$/
```

Comment:
Should the `.` be escaped? Perhaps you should use `str.EndsWith(hostname)` instead of a regexp

Reply:
`endsWith` would be simpler, thanks 👍 

 I'll open another PR to update this.

Reply:
Updated in https://github.com/vercel/next.js/pull/37206